ext/KrylovKitChainRulesCoreExt/KrylovKitChainRulesCoreExt.jl

```diff
@@ -11,5 +11,6 @@ include("utilities.jl")
 include("linsolve.jl")
 include("eigsolve.jl")
 include("svdsolve.jl")
+include("patch.jl")
```

Comment:
Thanks for the patch. I think you can just directly put the contents of "patch.jl" directly into "KrylovKitChainRulesCoreExt.jl"

Reply:
Perhaps rename it to `constructor.jl` for potential additions in the future.